stellar/deploy-contract.js

```diff
@@ -53,27 +53,25 @@ async function getInitializeArgs(config, chain, contractName, wallet, options) {
     }
 }
 
-async function processDeploy(options, config, chain) {
+async function deploy(options, config, chain) {
     const { contractName, privateKey, wasmPath } = options;
     const { rpc, networkType } = chain;
     const networkPassphrase = getNetworkPassphrase(networkType);
     const wallet = await getWallet(chain, options);
 
     if (!chain.contracts) {
         chain.contracts = {};
+        return;
     }
 
     const args = `--source ${privateKey} --rpc-url ${rpc} --network-passphrase "${networkPassphrase}"`;
     const cmd = `${stellarCmd} contract deploy --wasm ${wasmPath} ${args}`;
 
-    printInfo('Executing command', cmd);
-    printInfo('Deploying contract', contractName);
-
     let contractAddress = options.address;
```

Comment:
add a `prompt` to confirm the deployment, similar to evm/deploy-contract.js. Same for upgrade

Reply:
updated.